itests/org.openhab.binding.mqtt.homeassistant.tests/itest.bndrun

```diff
@@ -12,7 +12,9 @@ Import-Package: \
 	moquette-broker-[0-9.]*.jar;lib:=true
 
 -runrequires: \
-	bnd.identity;id='org.openhab.binding.mqtt.homeassistant.tests'
+	bnd.identity;id='org.openhab.binding.mqtt.homeassistant.tests',\
+	bnd.identity;id='org.graalvm.python.python-resources'
```

Comment:
This bundle was not properly resolved automatically. Seems to have been manually added previously for the test to work? This could be fixed by adding it to the bundle imports of one of the graalvm.python bundles in openhab-osgiify @ccutrer @HolgerHees.